### PR TITLE
Use connector class instead of task class

### LINKED ISSUE #38: Instantiate connectors instead of tasks
Currently, we're instantiating a task for each Conduit connector. We should consider mapping Conduit connectors to Kafka connectors, and not tasks.

Possible advantages are:
1. Most (all?) Kafka connector configs actually reference the connector class, and not the task class. Hence, this could make the migration to Conduit a bit smoother.
2. It's a higher level API.

### PR DESCRIPTION
**Pull request by hariso**
Resolves #38.

It's not possible to work with the `SourceConnector`/`SinkConnector` classes, since they do not expose methods for reading/writing data from sources/destinations. That means that we need to use the tasks internally.

However, Kafka connectors do expose the task class names, which makes it possible to still use connector classes as input.

A note: the configuration parameter is `wrapper.connector.class` and not `connector.class`, as in Kafka Connector. The reason is that `connector.class` is Kafka Connect's parameter, not a connector config. parameter.

### CODE REVIEW

File: scripts/dist.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
-./dist-no-libs.sh
-mkdir -p $TO_DIR/libs/
-cp -r libs/ $TO_DIR/libs/
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```
Comment (lovromazgon) [inline scripts/dist.sh:2]: Feels like these scripts change in every PR 😅

File: README.md [hunk 2]
```diff
@@ -41,29 +41,29 @@ a handshake with Conduit via standard output, and that is expected to be the fir
...
+|-----------------------------------------|----------------------------------------------------------------------------------------------|-----------------------------------------------------------------------|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `wrapper.connector.class`               | The class of the requested connector.                                                        | yes                                                                   | none    | `io.aiven.connect.jdbc.sink.JdbcSinkTask`                                                                                                                                                           |
```
Comment (lovromazgon) [inline README.md:46]: Shouldn't the example be changed now?

Comment (hariso) [reply README.md:46]: Good catch! The line is so long in the IDE that I didn't notice it. :d

File: scripts/dist.sh (hunk 1 above)
Comment (hariso) [reply scripts/dist.sh:2]: lol :D we're simply agile 😎

**Review by lovromazgon — APPROVED** [review]

**Pull request merged**